src/pages/contributors/jesse-glick.adoc

```diff
@@ -0,0 +1,107 @@
+Much of the work was arranging it so the program would keep running over controller restarts, which became the biggest technical hurdle.
+Eventually, I worked with Steven Connolly to create Multibranch Pipeline and Pipeline as code.
+This helped ensure our ultimate goal of keeping all these things within the same "package".
+
+Most recently, I've been working on the high availability system, and while it is a CloudBees proprietary feature, it has critical pieces that are rooted in open source such as the Pipeline continuing to run after controller restarts.
+Even now, there are challenges in keeping things running, working around limitations with the architecture, and dealing with isolation between the components.
+A lot of time is spent on these challenges, along with making sure tests keep running and making sure it's possible to do routine updates. 
+
+== Why choose Jenkins over other projects?
+
+There weren't a lot of options at the time, with Cruise Control being the only other alternative.
+It worked and was very extensible.
+Jenkins either did what we needed it to or we could add plugins that would perform the necessary functions.
+After becoming familiar with the system, extending Jenkins further was fairly straightforward.
+For instance, we wanted to generate a historical diff for quick reference that anyone could use.
+This wasn't possible inherently with Jenkins, but we were able to create a plugin that completed this particular task.
+Jenkins' versatility has only expanded since then, proving to be one of its greatest strengths.
+
+== What problems have you solved in Jenkins?
+
+I'm always looking for opportunities to simplify some parts of the architecture where possible.
+One project I worked on was the re-design of the Jenkins CLI.
+At the time, it was very troublesome from a security perspective and we would have to issue emergency advisories for something like the controller sending back something malicious.
+The project goal was to rip out all of the problematic functionality and turn it into a dump client.
+It could send a command and get back a stream of text, and the protocol was simplified so that it couldn't do anything more than these basic functions.
+However, this also meant tracking down every plugin that might interact with the CLI in this way and either updating code as needed or deprecating the function so that this would not persist.
+
+I also did some work on plugable storage changes, mainly the Artifact Manager on S3 plugin, to help externalize the storage of artifacts and statuses.
+The plugin ecosystem also forces you to work in particular ways, such as keeping track of what plugins might be affected by a change, or determining how a plugin might interact with various functions or other plugins. 
+It also pushes us to track down plugin maintainers for assistance with changes or updates, to ensure things are not broken by Jenkins core updates.
```

Comment:
```suggestion
I also did some work on plugable storage changes, mainly the Artifact Manager on the S3 plugin, to help externalize the storage of artifacts and statuses.
The plugin ecosystem also forces you to work in particular ways, such as keeping track of what plugins might be affected by a change, or determining how a plugin might interact with various functions or other plugins. 
It also pushes us to track down plugin maintainers for assistance with changes or updates, to ensure things are not broken by Jenkins core updates.
```

Reply:
The plugin name is Artifact Manager on S3, https://plugins.jenkins.io/artifact-manager-s3/. I'm happy to make this change for the grammar aspect, but it isn't correct according to the plugin page.

Reply:
My bad, thanks for your insight, Kevin. 👍

Reply:
No worries! I believe it came up at another point and I didn't clarify, so I figured I would do so and add a link to the plugin itself just to cover all bases.